glean-core/src/ping/mod.rs

```diff
@@ -107,12 +107,20 @@ impl PingMaker {
 
     fn get_ping_info(&self, glean: &Glean, storage_name: &str) -> JsonValue {
         let (start_time, end_time) = self.get_start_end_times(glean, storage_name);
-        json!({
+        let mut map = json!({
             "ping_type": storage_name,
             "seq": self.get_ping_seq(glean, storage_name),
             "start_time": start_time,
             "end_time": end_time,
-        })
+        });
+
+        // Get the experiment data, if available.
```

Comment:
What's the format of the experiment data in this section?

Reply:
I added the docs about the format in the relevant StorageManager function.